addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -634,7 +634,7 @@ async function toInline($editable, cssRules, $iframe) {
 async function flattenBackgroundImages(editable) {
     for (const backgroundImage of editable.querySelectorAll('*[style*=background-image]')) {
         if (backgroundImage.parentElement) { // If the image was nested, we removed it already.
-            const canvas = await html2canvas(backgroundImage);
+            const canvas = await html2canvas(backgroundImage, { scale: 1 });
```

Comment:
That's just GitHub making it weird because it shortens the link. This is the real commit message :
![image](https://user-images.githubusercontent.com/24205914/215804229-d90b0d78-53d6-490a-b108-08eefecc96ad.png)
But I can change it anyway if you want, no problem.